src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -127,7 +129,7 @@ class ServerListWindow final : public Window
                 Close();
                 break;
             case WIDX_PLAYER_NAME_INPUT:
-                WindowStartTextbox(*this, widgetIndex, STR_STRING, _playerName, 63);
+                WindowStartTextbox(*this, widgetIndex, STR_STRING, _playerName.c_str(), MaxPlayerNameLength);
```

Comment:
Interesting that the max length in this call used to be 63, while the max name length was 32 characters.